src/main/java/InfrastructureManager/Modules/NetworkStructure/Input/NetworkInput.java

```diff
@@ -0,0 +1,28 @@
+package InfrastructureManager.Modules.NetworkStructure.Input;
+
+import InfrastructureManager.ModuleManagement.Exception.Execution.ModuleExecutionException;
+import InfrastructureManager.Modules.NetworkStructure.Exception.NetworkModuleException;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import InfrastructureManager.ModuleManagement.ModuleInput;
+import InfrastructureManager.ModuleManagement.PlatformModule;
+import InfrastructureManager.Modules.NetworkStructure.Network;
+
+public class NetworkInput extends ModuleInput {
+	
+	private final Network network;
+	
+	public NetworkInput(PlatformModule module,String name,Network network) {
+		super(module,name);
+		this.network = network;
+	}
+    @Override
+    public String read() throws JsonProcessingException  {
```

Comment:
here only InterruptedException could be throw. JsonProcessingException or any other exception must be handled in this function.